spec/features/guide_history_spec.rb

```diff
@@ -59,3 +63,23 @@ def events
       .reverse
   end
 end
+
+
+RSpec.describe "Guide history", type: :feature do
+  scenario "viewing previous editions" do
+    guide = create(:published_guide)
+    guide.editions << build(:edition, version: 2)
+
+    visit edition_comments_path(guide.reload.latest_edition)
+
+    expect_edition_to_be_open("Version #2")
+
+    click_link "Version #1"
+
+    expect_edition_to_be_open("Version #1")
+  end
+
+  def expect_edition_to_be_open(title)
+    expect(page).to have_css(".open-edition .panel-heading", text: title)
```

Comment:
I meant that just testing for the title is enough in this test. Just realised that the title is displayed for all versions though so that's not really enough.

In a feature test you really want to be testing what actually happens to the page. Testing that something has a class isn't really enough. The test should be asserting that the content of that expanded section is actually visible.
If we deleted all the code inside that panel, in the view, this test would be green.